src/dynamic.ts

```diff
@@ -42,15 +42,15 @@ export default class DynamicThreadPool<
   }
 
   protected _chooseWorker (): WorkerWithMessageChannel {
-    let worker: WorkerWithMessageChannel
+    let worker: WorkerWithMessageChannel | undefined
     for (const entry of this.tasks) {
       if (entry[1] === 0) {
         worker = entry[0]
         break
       }
     }
 
-    if (worker) {
+    if (worker != null) {
```

Comment:
I would really prefer to have an helper method for all these null or undefined check. 

A file like utils.ts with: 

```
export function isNullOrUndefined<T>(object: T): boolean {
  return object == null;
}
```

And replacing such checks by:

`!isNullOrUndefined(worker)`
...


Or an Utils class containing only static methods as helpers.

Reply:
Looks like a duplicate review, I will resolve this one